Rust_implem/src/main.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(linked_list_cursors)]
+
+pub mod binomial_heap;
+
+fn main() {
+    println!("Hello, world!");
```

Comment:
Can you implement a little test ?

Reply:
You can find a test at the bottom of the main implementation file. The main file really is just so that cargo does not get confused. Maybe it could be used at some point but right now there is no need I think.

Reply:
ok mb